src/Plugin/QueueWorker/BulletinBase.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Drupal\govdelivery_bulletins\Plugin\QueueWorker;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+
+/**
+ * Provides base functionality for the GovDelivery Queue Workers.
+ */
+class BulletinBase extends QueueWorkerBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * BulletinBase constructor.
+   */
+  public function __construct() {
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+
```

Comment:
According to  docroot/core/lib/Drupal/Core/Plugin/ContainerFactoryPluginInterface.php  This has to have a return
```
   * @return static
   *   Returns an instance of this plugin.
   */
```

I am looking now for an example of what that ought to look like.

Reply:
Looks like something like this
```suggestion
    return new static(
      $configuration,
      $plugin_id,
      $plugin_definition,
      $container->get('plugin.manager.queue_worker')
    );
```

Reply:
Added the above, verbatim, @swirtSJW.....doesn't seem to add any additional hiccups.